src/App.js

```diff
@@ -93,6 +99,7 @@ class App extends Component {
             message={snackBarMessage}
             onClose={actions.closeSnackBar}
           />
+          {pathname !== '/' && <StaticAppBar />}
```

Comment:
Keep this logic inside StaticAppBar, would be more cleaner.

Reply:
@akshatnitd Yes, but instead of rendering the component and returning it null, this might be a better way.

Reply:
I agree. But, the difference is negligible. So, let's go for cleaner code 💯

Reply:
Now, it looks fine. But, once there would be multiple logic. We will need to anyways do it.

Reply:
@akshatnitd I tried what you are trying to say, after implmenting it, it is problematic keeping the current code in mind. If I put the logic in StaticAppBar, I am getting this error:

`Uncaught TypeError: Cannot read property 'offsetHeight' of undefined
    at StaticAppBar.componentDidMount (StaticAppBar.react.js:217)`

<img width="1439" alt="Screenshot 2019-06-03 at 7 44 31 PM" src="https://user-images.githubusercontent.com/18073126/58808636-0879c780-8638-11e9-9342-716f1a847e31.png">

Also, if we don't return anything, we will have to nest it into React.Fragment which is not desirable either.